src/core/server/app/middleware/graphqlBatch.ts

```diff
@@ -0,0 +1,97 @@
+import { RequestHandler, Response } from "express";
+
+import logger from "talk-server/logger";
+import { Request } from "talk-server/types/express";
+
+function wrapResponse(req: Request, res: Response) {
+  // If the request is not an array, or has no elements, we should skip it.
+  if (!Array.isArray(req.body) || req.body.length === 0) {
+    return res;
+  }
+
+  // If the request is an array, but it does not have an ID field, then we
+  // should skip it.
+  const needsUpgrade = Boolean(typeof req.body[0].id !== "undefined");
+  if (!needsUpgrade) {
```

Comment:
Upgrade usually means upgrading the connection to a websocket, maybe a more generic term like `skip` will do here too?

Reply:
Not that this is really important xD

Reply:
"needsUpgrade" to a relay batch connection :P the term "upgrade" isn't reserved here.